strawberryfields/backends/bosonicbackend/bosoniccircuit.py

```diff
@@ -301,23 +301,24 @@ def fidelity_coherent(self, alpha, modes=None):
         """ Returns a function that evaluates the Q function of the given state """
```

Comment:
I suspect this function should be hbar dependent, i.e., somewhere you need to check hbar in toder to know that q = \sqrt{2 hbar} Re(alpha) and p = \sqrt{2 hbar} Im(alpha)

Reply:
I will change alpha_mean